recording/docker-compose/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3"
+
+services:
+
+  nextcloud-talk-recording:
+    build:
+      context: ..
+      dockerfile: ./docker-compose/Dockerfile
+    shm_size: '2gb'
```

Comment:
It would be good to add `init: true` to solve the PID 1 zombie processes issue (now that `phusion/baseimage` is not being used).